mage_ai/frontend/pages/pipelines/[...slug].tsx

```diff
@@ -317,6 +317,36 @@ function PipelineDetailPage({
       ),
     },
   );
+
+  const [updatePipelineNoContent] = useMutation(api.pipelines.useUpdate(pipelineUUID));
+  const updatePipelineName = useCallback((name: string) => (
+    // @ts-ignore
+    updatePipelineNoContent({
+      pipeline: { name },
+    }).then((response: any) => {
+      onSuccess(
+        response, {
+          callback: ({ pipeline: { uuid } }) => {
+            router.push(`/pipelines/${uuid}`);
+            fetchFileTree();
+          },
+          onErrorCallback: ({
+            error: {
+              errors,
+              message,
+            },
+          }) => {
+            console.log(errors, message);
+          },
+        },
+      );
+    },
+  )), [
+    fetchFileTree,
+    updatePipelineNoContent,
+    router,
+  ]);
```

Comment:
is there a reason you did this in 2 separate functions vs just using one `useMutation`?

Reply:
I wasn't too familiar with this design pattern so I tried to emulate how it was done previously and did it separately to be safe. I can combine them

Reply:
Did you merge this before fixing?